app/src/main/java/org/brightmindenrichment/street_care/ui/community/CommunityHelpRequestFragment.kt

```diff
@@ -354,8 +372,19 @@ class CommunityHelpRequestFragment : Fragment() {
     ) {
         helpRequest.skills?.let { skills ->
             flexboxLayoutSkills.removeAllViews()
-            for(skill in skills) {
-                flexboxLayoutSkills.addView(createSkillTextView(skill, requireContext()))
+            for (skill in skills) {
+                val chip = Chip(requireContext()).apply {
+                    text = skill
+                    isClickable = true
+                    isCheckable = false
+
+                    // Set click listener for filtering events by skill
+                    setOnClickListener {
+                        bottomSheetBehavior.state = BottomSheetBehavior.STATE_COLLAPSED
+                        filterEventsBySkill(skill, helpRequest.id!!)
```

Comment:
Use a `let` block to prevent a NPE crash

`helpRequest.id?.let { it1 -> filterEventsBySkill(skill, it1) }`

Reply:
Addressed